app/helpers/application_helper.rb

```diff
@@ -225,9 +225,18 @@ def action_buttons(*args)
 
     content_tag(:div, :class => "btn-group") do
       primary + link_to(content_tag(:span, '', :class => 'caret'), '#', :class => "btn btn-default #{'btn-sm' if primary =~ /btn-sm/} dropdown-toggle", :'data-toggle' => 'dropdown') +
-      content_tag(:ul, :class => "dropdown-menu pull-right") do
-        args.map { |option| content_tag(:li, option) }.join(" ").html_safe
-      end
+        content_tag(:ul, :class => "dropdown-menu pull-right") do
```

Comment:
One idea here: In core we have `link_to_if_authorized` method, maybe we should also update this method to have same behavior.

Reply:
Good idea.
Should I create a new PR for this?

Reply:
So I found that `link_to_if_authorized` already check if link is authorized, if not it will display link as disabled